plugins/woocommerce/tests/e2e-pw/tests/merchant/create-variable-product.spec.js

```diff
@@ -64,6 +64,7 @@ test.describe.serial( 'Add New Variable Product Page', () => {
 				'val1 | val2'
 			);
 		}
+		await page.keyboard.press( 'ArrowUp' );
```

Comment:
We need to call this event because the Playwright method `page.fill` is not triggering the keyboard event.